### PR TITLE
Fix spanish translation

### PR DESCRIPTION
**Pull request by jonathanhecl**

### CODE REVIEW

File: server_release_template/web_gui/js/app/nls/es/settings.js [hunk 1]
```diff
@@ -210,8 +210,8 @@ Una resolución inferior al 100% puede reducir la latencia y mejorar la calidad
     "_root_connection_frameQueueSize.name": "Tamaño de la cola de cuadros",
     "_root_connection_frameQueueSize.description": "Tamaño máximo de la cola de fotogramas utilizado por el cliente.",
     "_root_connection_aggressiveKeyframeResend.name": "Reenvía los fotogramas clave de forma agresiva",
-    "_root_connection_aggressiveKeyframeResend.description": `Reducir el intervalo de reenvío de los cuadros de tipo 'I' de 100ms a 5ms.
```
Comment (zarik5) [inline server_release_template/web_gui/js/app/nls/es/settings.js:213]: It's was correct specifying "I" frames.

File: server_release_template/web_gui/js/app/nls/es/settings.js [hunk 2]
```diff
@@ -210,8 +210,8 @@ Una resolución inferior al 100% puede reducir la latencia y mejorar la calidad
...
-Sólo se utiliza cuando se detecta pérdida de paquetes. Mejora la experiencia visual en caso de pérdida de paquetes.`,
+    "_root_connection_aggressiveKeyframeResend.description": `Reducir el intervalo de reenvío de cuadros de 100ms a 5ms.
+Sólo se utiliza cuando se detecta una pérdida de paquetes. Mejora la experiencia visual en caso de pérdida de paquetes.`,
```
Comment (zarik5) [inline server_release_template/web_gui/js/app/nls/es/settings.js:214]: It was correct specifying "I" frames.

**Review by zarik5 — CHANGES_REQUESTED** [review]

File: server_release_template/web_gui/js/app/nls/es/settings.js (hunk 2 above)
Comment (jonathanhecl) [reply server_release_template/web_gui/js/app/nls/es/settings.js:214]: I don't understand what type "I" frames are, besides in English it doesn't say that.

Comment (jonathanhecl) [reply server_release_template/web_gui/js/app/nls/es/settings.js:214]: I suppose they are type "key" frames, no "I".

Comment (zarik5) [reply server_release_template/web_gui/js/app/nls/es/settings.js:214]: "I" is the technical term for keyframes, in the context of h264/HEVC encoding.

**Pull request merged**